react/features/welcome/components/AbstractWelcomePage.js

```diff
@@ -269,16 +274,13 @@ export class AbstractWelcomePage extends Component<Props, *> {
  *
  * @param {Object} state - The redux state.
  * @protected
- * @returns {{
- *     _calendarEnabled: boolean,
- *     _room: string,
- *     _settings: Object
- * }}
+ * @returns {Props}
  */
 export function _mapStateToProps(state: Object) {
     return {
         _calendarEnabled: isCalendarEnabled(state),
         _enableInsecureRoomNameWarning: state['features/base/config'].enableInsecureRoomNameWarning || false,
+        _moderatedRoomServiceUrl: state['features/base/config'].moderatedRoomServiceUrl,
```

Comment:
Maybe add that to config.js commented with some explanation of how it is used.

Reply:
Thanks, added!